src/api.ts

```diff
@@ -16,7 +15,6 @@ function get<T>(url: string): Promise<T> {
   // eslint-disable-next-line prefer-template
```

Comment:
The ESLint rule `prefer-template` is disabled here. Consider using template literals for constructing the `fullURL` to adhere to modern JavaScript practices.